app/src/main/java/com/cornellappdev/android/eatery/MainListAdapter.java

```diff
@@ -237,16 +276,31 @@ public void onClick(View v) {
       int adapterPosition = getAdapterPosition();
       mListAdapterOnClickHandler.onClick(adapterPosition, cafeListFiltered);
     }
+
   }
 
   private static String convertName(String str) {
-    if (str.equals("104West!.jpg")) return "104-West.jpg";
-    if (str.equals("McCormick's.jpg")) return "mccormicks.jpg";
-    if (str.equals("Franny's.jpg")) return "frannys.jpg";
-    if (str.equals("Ice Cream Cart.jpg")) return "icecreamcart.jpg";
-    if (str.equals("Risley Dining Room.jpg")) return "Risley-Dining.jpg";
-    if (str.equals("Martha's Express.jpg")) return "Marthas-Cafe.jpg";
-    if (str.equals("Bus Stop Bagels.jpg")) return "Bug-Stop-Bagels.jpg";
+    if (str.equals("104West!.jpg")) {
+      return "104-West.jpg";
+    }
+    if (str.equals("McCormick's.jpg")) {
+      return "mccormicks.jpg";
+    }
+    if (str.equals("Franny's.jpg")) {
+      return "frannys.jpg";
+    }
+    if (str.equals("Ice Cream Cart.jpg")) {
+      return "icecreamcart.jpg";
+    }
+    if (str.equals("Risley Dining Room.jpg")) {
+      return "Risley-Dining.jpg";
+    }
+    if (str.equals("Martha's Express.jpg")) {
+      return "Marthas-Cafe.jpg";
+    }
+    if (str.equals("Bus Stop Bagels.jpg")) {
+      return "Bug-Stop-Bagels.jpg";
+    }
```

Comment:
Not sure what's unreadable about the original snippet, but this should be changed into a `case` statement instead.

Reply:
This was fixed automatically by Google linting. Often times changes like this I didn't do manually, the original just violated the style guide (I observed you were using Google, I believe).

Reply:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used